README.md

```diff
@@ -104,9 +104,15 @@ The following section lists the available commands for the plugin and their expl
 
    Lists services and their running state. If a `service` is specified only the status of that service is displayed. If no service is provided then only supported orchestrators are reported.
 
-6. `vagrant service-manager restart [service]`
+6. `vagrant service-manager [operation] [service]`
 
-   Restarts the given service in the box.
+   where `operation` could be:
+
+   * `restart` : Restart the given service in the box.
+   * `start`   : Start the given service in the box.
+   * `stop`    : Stop the given service in the box.
+
+   and `service` could be `docker`, `openshift`, `kubernetes` or any systemd service.
```

Comment:
> I hope you mean to say. Let's go ahead with this PR merge and later on 
implement based on ML discussion on systemd service.

-1 It is always harder to remove something than adding it. I see me reason 
to add it, if there is a chance we remove it later. In order to merge I'd 
remove any mention of systemd services. Then if we want to go ahead and use 
the plugin also to control systemd services we can add the appropriate 
docs.